newIDE/app/src/fixtures/GDevelopServicesTestData.js

```diff
@@ -267,7 +267,7 @@ export const erroredCordovaBuild: Build = {
   status: 'error',
   logsKey: '/fake-error.log',
   createdAt: 1515084391000,
-  updatedAt: 1515084399000,
+  updatedAt: Date.now(),
```

Comment:
Is this necessary?

Reply:
In BuildsList, there is some additional logic that hides any action on older builds. Since I wanted to show their "true" self, I changed the updatedAt values